docs/source/conf.py

```diff
@@ -190,3 +190,5 @@ def setup(app):
     app.add_js_file('js/custom.js')
 
 # -- Extension configuration -------------------------------------------------
+# Napoleon settings
+napoleon_use_ivar = True
```

Comment:
What does this do exactly ?

Reply:
This tunes the rendering of "Attributes" list (see for example class: DatasetInfo).

Default value is `False`, and in that case each attribute is rendered with a format analog to class methods.

Otherwise, if set to `True`, Attributes are rendered with the same format as Args (Parameters).

Reply:
Cool !